src/containers/CellBoarderAndFlagManager.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file CellBoarderAndFlagManager.h
+ * @author seckler
+ * @date 14.05.18
+ */
+
+#pragma once
+
+namespace autopas {
+/**
+ * interface class to handle cell boarders and cell types of cells.
+ * @todo: add cell boarder handling
+ */
+class CellBoarderAndFlagManager {
```

Comment:
Cell~Boarder~AndFlagManager -> CellBorderAndFlagManager ?

Reply:
to board vs border vs bored -- English at its best :)